client/lib/paths/login/index.js

```diff
@@ -42,6 +42,7 @@ export function login( {
 	usernameOnly = undefined,
 	gravatarFrom = undefined,
 	gravatarFlow = undefined,
+	plugin_name = undefined,
```

Comment:
Nit: naming convention, also need to change in all its uses if agreed
```suggestion
	pluginName = undefined,
```